### ISSUE TITLE
I don't understand how to use isValidated() method

### ISSUE THREAD

**Issue by ykptke** [newbreedofgeek/react-stepzilla#1]
I can't access refs.component isValidated() method from step component. please help me

**Comment by newbreedofgeek** [comment]
@ykptke sorry my example code in the src/examples directory is bad. I will post a better example soon.

But in the Step component, if you need validation then you should expose a isValidated() method that return true (if form input is valid) or false (if form input is not valid).

StepZilla will check for the existence of a isValidated() method is a Step component, if found it then will validate the component before going to the next.

So basically you don't need to access refs.component isValidated() yourself. 

Give me a few hours and i'll post a complete example.

**Comment by ykptke** [comment]
@newbreedofgeek thanks. :+1:  I look forward to your example

**Comment by newbreedofgeek** [comment]
@ykptke I pushed 1.7.0 (4182509 ) to master and npm. It has a complete working example in `src/examples`

You can run it in the browser like so:
- run `npm install`
- then run `npm run example`
- then go to `http://localhost:8080/webpack-dev-server/src/examples/index.html` in your browser

Hope this helps you.
